Cannot find module '../lib/load-env-config'.

I use a similar structure of [this example](https://github.com/zeit/next.js/tree/canary/examples/with-firebase-hosting-and-typescript), during compilation of functions:
```
tsc --project src/functions
```

I got this error:
```
node_modules/next/types/index.d.ts:8:21 - error TS2307: Cannot find module '../lib/load-env-config'.
```

## System information

- OS: (K)Ubuntu 19.10
- Version of Node: 10.19.0
- Version of Next.js: 9.3.2

--- a/packages/next/types/index.d.ts
+++ b/packages/next/types/index.d.ts
@@ -5,7 +5,7 @@
 import React from 'react'
 import { ParsedUrlQuery } from 'querystring'
 import { IncomingMessage, ServerResponse } from 'http'
-import { Env } from '../lib/load-env-config'
+import { Env } from '../dist/lib/load-env-config'
 
 import {
   NextPageContext,

